scylla-macros/src/serialize/value.rs

```diff
@@ -96,13 +99,88 @@ pub(crate) fn derive_serialize_value(
 ) -> Result<syn::ItemImpl, syn::Error> {
     let input: syn::DeriveInput = syn::parse(tokens_input)?;
     let struct_name = input.ident.clone();
-    let named_fields = crate::parser::parse_named_fields(&input, "SerializeValue")?;
     let (impl_generics, ty_generics, where_clause) = input.generics.split_for_impl();
     let attributes = Attributes::from_attributes(&input.attrs)?;
 
     let crate_path = attributes.crate_path();
     let implemented_trait: syn::Path = parse_quote!(#crate_path::SerializeValue);
 
+    if attributes.transparent {
+        if let syn::Data::Struct(data) = &input.data {
+            if data.fields.len() != 1 {
+                return Err(syn::Error::new_spanned(
+                    data.fields.clone(),
+                    "#[scylla(transparent)] requires exactly one field",
+                ));
+            }
+
+            let field = data.fields.iter().next().unwrap();
```

Comment:
In deserialization you extracted transparent case into a separate function. Can you do the same here?